{"errorName":"Error","exitCode":1,"exitSignal":null,"level":"error","message":"Erro na execução do script 20segundos.py","requestId":"b52f6014-fc7c-48ca-8c15-4b42d2c57654","scriptId":"20segundos0","stackTrace":"Error: NameError: name 'joao' is not defined\n    at PythonShell.parseError (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:303:21)\n    at terminateIfNeeded (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:193:32)\n    at ChildProcess.<anonymous> (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:185:13)\n    at ChildProcess.emit (node:events:511:28)\n    at ChildProcess._handle.onexit (node:internal/child_process:293:12)\r\n    ----- Python Traceback -----\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 22, in <module>\r\n      main()\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 9, in main\r\n      print(joao)\r\n            ^^^^","timestamp":"2024-01-06 11:35:11"}
{"errorName":"Error","exitCode":1,"exitSignal":null,"level":"error","message":"Erro na execução do script 20segundos.py","requestId":"27c54606-42e7-4e04-916b-2e3514443379","scriptId":"20segundos0","stackTrace":"Error: NameError: name 'joao' is not defined\n    at PythonShell.parseError (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:303:21)\n    at terminateIfNeeded (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:193:32)\n    at ChildProcess.<anonymous> (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:185:13)\n    at ChildProcess.emit (node:events:511:28)\n    at ChildProcess._handle.onexit (node:internal/child_process:293:12)\r\n    ----- Python Traceback -----\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 22, in <module>\r\n      main()\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 9, in main\r\n      print(joao)\r\n            ^^^^","timestamp":"2024-01-06 11:37:53"}
{"errorName":"Error","exitCode":1,"exitSignal":null,"level":"error","message":"Erro na execução do script 20segundos.py","requestId":"faa6caff-1d35-4177-8594-0374362eab53","scriptId":"20segundos0","stackTrace":"Error: NameError: name 'joao' is not defined\n    at PythonShell.parseError (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:303:21)\n    at terminateIfNeeded (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:193:32)\n    at ChildProcess.<anonymous> (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:185:13)\n    at ChildProcess.emit (node:events:511:28)\n    at ChildProcess._handle.onexit (node:internal/child_process:293:12)\r\n    ----- Python Traceback -----\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 22, in <module>\r\n      main()\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 9, in main\r\n      print(joao)\r\n            ^^^^","timestamp":"2024-01-06 11:43:48"}
{"errorName":"Error","exitCode":1,"exitSignal":null,"level":"error","message":"Erro na execução do script 20segundos.py","requestId":"a1204d85-bbb5-484f-a158-a11220fa6258","scriptId":"20segundos0","stackTrace":"Error: NameError: name 'joao' is not defined\n    at PythonShell.parseError (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:303:21)\n    at terminateIfNeeded (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:193:32)\n    at ChildProcess.<anonymous> (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:185:13)\n    at ChildProcess.emit (node:events:511:28)\n    at ChildProcess._handle.onexit (node:internal/child_process:293:12)\r\n    ----- Python Traceback -----\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 22, in <module>\r\n      main()\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 9, in main\r\n      print(joao)\r\n            ^^^^","timestamp":"2024-01-06 11:43:51"}
{"errorName":"Error","exitCode":1,"exitSignal":null,"level":"error","message":"Erro na execução do script 20segundos.py","requestId":"2cc8e039-d646-4531-95d0-094006a3671e","scriptId":"20segundos0","stackTrace":"Error: NameError: name 'joao' is not defined\n    at PythonShell.parseError (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:303:21)\n    at terminateIfNeeded (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:193:32)\n    at ChildProcess.<anonymous> (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:185:13)\n    at ChildProcess.emit (node:events:511:28)\n    at ChildProcess._handle.onexit (node:internal/child_process:293:12)\r\n    ----- Python Traceback -----\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 22, in <module>\r\n      main()\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 9, in main\r\n      print(joao)\r\n            ^^^^","timestamp":"2024-01-06 11:43:53"}
{"errorName":"Error","exitCode":1,"exitSignal":null,"level":"error","message":"Erro na execução do script 20segundos.py","requestId":"1e7702e8-dafb-4b7f-b821-b9303496eb86","scriptId":"20segundos0","stackTrace":"Error: NameError: name 'joao' is not defined\n    at PythonShell.parseError (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:303:21)\n    at terminateIfNeeded (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:193:32)\n    at ChildProcess.<anonymous> (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:185:13)\n    at ChildProcess.emit (node:events:511:28)\n    at ChildProcess._handle.onexit (node:internal/child_process:293:12)\r\n    ----- Python Traceback -----\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 22, in <module>\r\n      main()\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 9, in main\r\n      print(joao)\r\n            ^^^^","timestamp":"2024-01-06 11:44:14"}
{"errorName":"Error","exitCode":1,"exitSignal":null,"level":"error","message":"Erro na execução do script 20segundos.py","requestId":"d5cc4033-c1d3-49f4-854f-f961180ef0c4","scriptId":"20segundos0","stackTrace":"Error: NameError: name 'joao' is not defined\n    at PythonShell.parseError (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:303:21)\n    at terminateIfNeeded (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:193:32)\n    at ChildProcess.<anonymous> (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:185:13)\n    at ChildProcess.emit (node:events:511:28)\n    at ChildProcess._handle.onexit (node:internal/child_process:293:12)\r\n    ----- Python Traceback -----\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 22, in <module>\r\n      main()\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 9, in main\r\n      print(joao)\r\n            ^^^^","timestamp":"2024-01-06 11:44:16"}
{"errorName":"Error","exitCode":1,"exitSignal":null,"level":"error","message":"Erro na execução do script 20segundos.py","requestId":"cc118ead-1aad-4999-92ff-c2fbd87aaf81","scriptId":"20segundos0","stackTrace":"Error: NameError: name 'joao' is not defined\n    at PythonShell.parseError (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:303:21)\n    at terminateIfNeeded (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:193:32)\n    at ChildProcess.<anonymous> (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:185:13)\n    at ChildProcess.emit (node:events:511:28)\n    at ChildProcess._handle.onexit (node:internal/child_process:293:12)\r\n    ----- Python Traceback -----\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 22, in <module>\r\n      main()\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 9, in main\r\n      print(joao)\r\n            ^^^^","timestamp":"2024-01-06 11:44:18"}
{"errorName":"Error","exitCode":1,"exitSignal":null,"level":"error","message":"Erro na execução do script 20segundos.py","requestId":"a22f3f66-2350-4a0e-abd8-71ef5d48bc3d","scriptId":"20segundos0","stackTrace":"Error: NameError: name 'joao' is not defined\n    at PythonShell.parseError (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:303:21)\n    at terminateIfNeeded (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:193:32)\n    at ChildProcess.<anonymous> (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:185:13)\n    at ChildProcess.emit (node:events:511:28)\n    at ChildProcess._handle.onexit (node:internal/child_process:293:12)\r\n    ----- Python Traceback -----\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 22, in <module>\r\n      main()\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 9, in main\r\n      print(joao)\r\n            ^^^^","timestamp":"2024-01-06 11:48:43"}
{"errorName":"Error","exitCode":1,"exitSignal":null,"level":"error","message":"Erro na execução do script 20segundos.py","requestId":"668479c1-94af-4c29-99bb-a9ffc6858330","scriptId":"20segundos0","stackTrace":"Error: NameError: name 'joao' is not defined\n    at PythonShell.parseError (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:303:21)\n    at terminateIfNeeded (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:193:32)\n    at ChildProcess.<anonymous> (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:185:13)\n    at ChildProcess.emit (node:events:511:28)\n    at ChildProcess._handle.onexit (node:internal/child_process:293:12)\r\n    ----- Python Traceback -----\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 22, in <module>\r\n      main()\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 9, in main\r\n      print(joao)\r\n            ^^^^","timestamp":"2024-01-06 11:49:03"}
{"errorName":"Error","exitCode":1,"exitSignal":null,"level":"error","message":"Erro na execução do script 20segundos.py","requestId":"36665dd0-ac7d-483b-92f2-355190cbe068","scriptId":"20segundos0","stackTrace":"Error: NameError: name 'joao' is not defined\n    at PythonShell.parseError (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:303:21)\n    at terminateIfNeeded (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:193:32)\n    at ChildProcess.<anonymous> (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:185:13)\n    at ChildProcess.emit (node:events:511:28)\n    at ChildProcess._handle.onexit (node:internal/child_process:293:12)\r\n    ----- Python Traceback -----\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 22, in <module>\r\n      main()\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\20segundos.py\", line 9, in main\r\n      print(joao)\r\n            ^^^^","timestamp":"2024-01-06 11:52:48"}
{"errorName":"Error","exitCode":1,"exitSignal":null,"level":"error","message":"Erro na execução do script smtester.py","requestId":"f098a57f-6505-4f81-a1f8-c94a723e6387","scriptId":"smtester0","stackTrace":"Error: NameError: name 'loads' is not defined. Did you mean: 'locals'?\n    at PythonShell.parseError (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:303:21)\n    at terminateIfNeeded (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:193:32)\n    at ChildProcess.<anonymous> (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:185:13)\n    at ChildProcess.emit (node:events:511:28)\n    at ChildProcess._handle.onexit (node:internal/child_process:293:12)\r\n    ----- Python Traceback -----\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\smtester.py\", line 379, in <module>\r\n      loop.run_until_complete(main())\r\n    File \"C:\\Users\\joaop\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\asyncio\\base_events.py\", line 653, in run_until_complete\r\n      return future.result()\r\n             ^^^^^^^^^^^^^^^\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\smtester.py\", line 325, in main\r\n      data = loads(body)\r\n             ^^^^^","timestamp":"2024-01-07 13:44:55"}
{"errorName":"Error","exitCode":1,"exitSignal":null,"level":"error","message":"Erro na execução do script smtester.py","requestId":"f098a57f-6505-4f81-a1f8-c94a723e6387","scriptId":"smtester1","stackTrace":"Error: NameError: name 'loads' is not defined. Did you mean: 'locals'?\n    at PythonShell.parseError (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:303:21)\n    at terminateIfNeeded (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:193:32)\n    at ChildProcess.<anonymous> (C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\node_modules\\python-shell\\index.js:185:13)\n    at ChildProcess.emit (node:events:511:28)\n    at ChildProcess._handle.onexit (node:internal/child_process:293:12)\r\n    ----- Python Traceback -----\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\smtester.py\", line 379, in <module>\r\n      loop.run_until_complete(main())\r\n    File \"C:\\Users\\joaop\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\asyncio\\base_events.py\", line 653, in run_until_complete\r\n      return future.result()\r\n             ^^^^^^^^^^^^^^^\r\n    File \"C:\\Users\\joaop\\OneDrive\\Documents\\GitHub\\code-runner\\codes\\smtester.py\", line 325, in main\r\n      data = loads(body)\r\n             ^^^^^","timestamp":"2024-01-07 13:44:55"}
